build-system/tasks/pr-deploy-bot-utils.js

```diff
@@ -70,13 +70,17 @@ async function replaceUrls(dir) {
 }
 
 async function signalPrDeployUpload(result) {
+  // TODO(rsimha): Remove this check once Travis is shut down.
+  if (!isCircleciBuild()) {
+    return;
+  }
   const loggingPrefix = getLoggingPrefix();
   logWithoutTimestamp(
     `${loggingPrefix} Reporting`,
     cyan(result),
     'to the pr-deploy GitHub App...'
   );
-  const sha = gitCommitHash();
+  const sha = shortSha(ciPullRequestSha());
   const ciBuild = ciBuildId();
   const baseUrl = 'https://amp-pr-deploy-bot.appspot.com/v0/pr-deploy/';
   const url = `${baseUrl}cibuilds/${ciBuild}/headshas/${sha}/${result}`;
```

Comment:
Since ciBuild won't be needed anymore, we can change the url to `${baseUrl}headshas/${sha}/${result}` and make the corresponding change to the bot's router

Reply:
Good catch. Done here, and also in https://github.com/ampproject/amp-github-apps/pull/1201.